src_rebuild/Game/C/wheelforces.c

```diff
@@ -528,6 +528,32 @@ void AddWheelForcesDriver1(CAR_DATA* cp, CAR_LOCALS* cl)
 	else
 	{
 		cp->hd.wheel_speed = cdz / 64 * (cl->vel[2] / 64) + cdx / 64 * (cl->vel[0] / 64);
+
+		// [A] wibbly wobbly fuckery hack...
+		if (car_cos->extraInfo & 4)
+		{
+			if (cp->thrust == 0 && cp->handbrake && (cp->hd.speed > 0 && cp->hd.speed < 4))
+			{
+				cp->hd.speed--;
+				cp->hd.wheel_speed >>= 1;
```

Comment:
I've attempted to make the slowing down effect as gradual as possible by not merely setting these values to zero. However, removing these steps entirely will result in a very bizarre effect that looks like the truck being sucked into a wormhole despite not getting anywhere.. it's quite funny but also very annoying, lol.